src/components/collapsible/CollapsibleConnected.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import {connect} from 'react-redux';
+import {findWhere} from 'underscore';
+
+import {IReactVaporState} from '../../ReactVapor';
+import {IDispatch, ReduxUtils} from '../../utils/ReduxUtils';
+import {
+    Collapsible,
+    CollapsibleDispatchProps,
+    CollapsibleOwnProps,
+    CollapsibleProps,
+    CollapsibleStateProps,
+} from './Collapsible';
+import {addCollapsibleContainer, removeCollapsibleContainer, setCollapsibleExpanded} from './CollapsibleActions';
+
+const mapStateToProps = (state: IReactVaporState, ownProps: CollapsibleOwnProps): CollapsibleStateProps => {
+    const collapsibleState = findWhere(state.collapsibles, {id: ownProps.id});
+
+    return {expanded: collapsibleState && collapsibleState.expanded};
+};
+
+const mapDispatchToProps = (
+    dispatch: IDispatch,
+    ownProps: CollapsibleOwnProps,
+): CollapsibleDispatchProps => ({
+    onMount: () => dispatch(addCollapsibleContainer(ownProps.id, !!ownProps.expandedOnMount)),
+    onUnmount: () => dispatch(removeCollapsibleContainer(ownProps.id)),

```

Comment:
You could use the other actions that are not deprecated instead

Reply:
Thx forgot to do this